app/src/helpers/windowHelpers.ts

```diff
@@ -198,12 +193,12 @@ export function hideWindow(
 }
 
 export function injectCSS(browserWindow: BrowserWindow): void {
-  if (!shouldInjectCSS()) {
+  const cssToInject = getCSSToInject();
+
+  if (!cssToInject) {
     return;
   }
```

Comment:
This could be tested too along with your other `will not inject css ...` tests

Reply:
We should be in `will not inject if getCSSToInject is empty`

Reply:
> We should be in `will not inject if getCSSToInject is empty`

Oh, missed this one. 👍.